xml/ha_acl.xml

```diff
@@ -52,6 +52,8 @@
    CIB version, refer to the <citetitle>Access Control List</citetitle>
    chapter in the <citetitle>&haguide;</citetitle> for &productname; 11 SP3 or earlier. It is
    available from <link xlink:href="http://www.suse.com/documentation/"/>.
+  <remark>taroth 2018-03-02: ygao, did we have any pre-pacemaker 1.0 CIB
+   versions in SLE 11*? probably not</remark>
```

Comment:
No, we shipped pacemaker-1.0.1 already in SLE11 GA.

Reply:
OK, removed the remark.